gvm/protocols/ospv1.py

```diff
@@ -247,7 +252,8 @@ def stop_scan(self, scan_id=None):
         """Stop a currently running scan.
 
         Args:
-            scan_id (uuid): Identifier for a running scan.
+            scan_id (str, optional): UUID identifier for a running scan.
```

Comment:
scan_id for stop_scan is not optional.

Reply:
In that case the method signature must be changed. I'll update it accordingly.